src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -149,90 +149,61 @@ them, and how to use them to configure your environment.
-<prop key="proactive.communication.protocol" value="http" />
-<prop key="proactive.http.port" value="PORT" />
-<prop key="proactive.net.noprivate" value="true" />
-<prop key="proactive.useIPaddress" value="true" />
-```
-
-Note that you need to specify a different port for both configurations.
-Sample configurations can be found in */samples/ec2*.
+    
+    -   *operatingSystem:* The operating system running in the AWS instance.
+    
+    -   *downloadCommand:* The command to download the Proactive node.jar. This command is executed in all the newly created AWS instances at cloud init time. 
+    	The URL needs to be accessible from the AWS cloud.
+    	
+    -   *additionalProperties:* Additional java command properties to be added when connecting the node.jar to the resource manager (e.g. \"-Dpropertyname=propertyvalue\").
+     
+    -   *ram:* The desired RAM amount expressed in Mega Bytes for each AWS instance that needs to be created.
+    
+    -   *cpu:* The desired CPU amount amount expressed in units for each AWS instance that needs to be created.
+    
++
+WARNING: If the combination between RAM and CPU does not match any existing AWS instance type, then the closest to the specified parameters will be selected.
++
+    
 
 Using this configuration, you can start a Resource Manager and a
 Scheduler using the */bin/proactive-server* script. An EC2 NodeSource can
-now be added using the command line interface of the Resource Manager:
+now be added using the *Create Node Source* panel in the Resource Manager or the command line interface:
 
-    $ PROACTIVE_HOME/bin/proactive-client --createns ec2 -infrastructure org.ow2.proactive.resourcemanager.nodesource.infrastructure.EC2Infrastructure ./config/rm/deployment/ec2.properties rmURL RMCredentialsPath nodeHttpPort
+    $ PROACTIVE_HOME/bin/proactive-client --createns ec2 -infrastructure org.ow2.proactive.resourcemanager.nodesource.infrastructure.AWSEC2Infrastructure aws_key aws_secret_key rmDomain connectorIaasURL image numberOfInstances numberOfNodesPerInstance operatingSystem downloadCommand additionalProperties ram cpu 
 
-You will need to specify:
 
--   **configurationFile** - EC2 configuration file such as
-    */config/rm/deployment/ec2.properties*, which you should have filled
-    with your AWS credentials beforehand.
+Once the Resource Manager is up and running with its EC2 NodeSource, new
+jobs can be added by using */bin/proactive-client*, or the Scheduler
+GUI.
 
--   **rmURL** - a fully qualified URL conforming to
-    *protocol://IP:port/*, where IP is the public IP of the machine
-    where the Resource Manager is launched. Protocol and port are
-    mandatory, as these values will be used to configure the ProActive
-    runtime of the EC2 instances at boot. It describes the protocol and
-    port used for communications from the instance to the Resource
-    Manager.
+When the Scheduler is stopped normally, all the AWS instances created will be terminated. 
```

Comment:
You mean _terminated_ in the terms of Amazon? if I remember correctly, there is a difference between stop and terminate. With the former, the attached bootable EBS volume will not be deleted:

http://support.rightscale.com/09-Clouds/AWS/FAQs/FAQ_0149_-_What%27s_the_difference_between_Terminating_and_Stopping_an_EC2_Instance%3F/index.html

I think it is really important to give more information here.